### DISCUSSION TITLE
1password student pack error

### DISCUSSION THREAD

**Discussion by nooma-42** [community/community#65364] [GitHub Education] [answered]
### Hi everyone,

I signed in to 1password with github account. and get following error, so I can't apply student free trial 1 yr. How to go about this?
```
Something went wrong.

An unexpected error occurred.

ServerError: 400 (Could not find the partner activation code.)

Session: OFT32NY4WJE2PCYUYZM4GHCKZA

Time: 2023-08-29T16:07:12.678Z
```

**Comment by arafajs** [comment]
This can happen for a few reasons:

- You may have entered the activation code incorrectly.
- The activation code may have expired.
- The activation code may be invalid.

To troubleshoot this issue, please try the following:

- Check that you have entered the activation code correctly.
- Make sure that the activation code has not expired. The student pack activation codes expire 1 year after they are issued.
- If you are still having trouble, contact 1Password support for help.

**Reply by nooma-42** [reply]
I don't have to (and can't) enter activation code when signing up with github. It's automatic process.

**Comment by nooma-42** [comment, marked as answer]
I've solved this issue, here's the response from support

The error you're seeing suggests to me that you're attempting to redeem the promotion outside the region it is assigned to. For example, if the promotion is claimed for "[1password.com](http://1password.com/)" it cannot be used for 1Password accounts on "[1password.eu](http://1password.eu/)".

No worries, though, as this should be an easy fix! You'll need to go to [the offer page](https://1password.com/developers/students) and click "Sign in with GitHub" as you previously did. Once you've authorized your account on GitHub you'll be taken to a page titled "Just one more step..." where you'll see an option to "Change region". Importantly, you'll need to make that the region chosen here is the same region where your 1Password account exists/will be created. For example, if you sign into 1Password using "[1password.eu](http://1password.eu/)", the region here must also be "[1password.eu](http://1password.eu/)". This should allow you to correctly redeem the promotion.

**Reply by Mehdy922** [reply]
The same is happening for me aswell.. I went to the offer page and clicked sing in with github but there is no such thing as just one more step it takes me straight to that error page.
